roles/test-win-winlogbeat-file/tasks/main.yml

```diff
@@ -4,13 +4,13 @@
 - name: Install Winlogbeat service
   script: run_script.ps1 -script {{ installdir }}\\install-service-{{ beat_name }}.ps1
 
-- name: Disable elasticsearch output and enable file output
+- name: Replace configuration file with template (windows)
+  win_copy: src={{beat_name}}.yml dest={{installdir}}
+
+- name: Replace template variables
   script: replace.ps1 -file {{ installdir }}\\winlogbeat.yml -find "{{ item.find | quote }}" -replace "{{ item.replace | quote }}"
   with_items:
-    - {find: '(?m)(^\s*)(elasticsearch:)', replace: '$1#$2'}
-    - {find: '(?m)(^\s*)(hosts:.*)', replace: '$1#$2'}
-    - {find: '(?m)(^\s*)#(file:.*)', replace: '$1$2'}
-    - {find: '(?s)(\s+file:.*)#(path:)[^\n]*', replace: '$1$2 {{ winlogbeat_data_dir }}/output'}
+    - {find: '(?s)\{\{ winlogbeat_data_dir \}\}', replace: '{{ winlogbeat_data_dir }}'}
```

Comment:
I tried `win_template` first. Sadly it didn't work and this was my fallback.

`fatal: [tester-win12-64] => module copy not found in configured module paths.  Additionally, core modules are missing. If this is a checkout, run 'git submodule update --init --recursive' to correct this problem.`

I'll try it again and debug it a little further. I was using the hetzner machine and was hesitant to change ansible.